src/css/tickets.css

```diff
@@ -0,0 +1,120 @@
+/*Tickets page CSS*/
+
+
+ /* announcements bar */
+.announcements-container {
+  display: flex;
+  align-items: center;
+  background-color: rgb(231, 211, 31);
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.announcements-container > div {
+  background-color: rgb(231, 211, 31);
+  color: rgb(0, 0, 0);
+  margin: 0px;
+  text-align: left;
+  line-height: 25px;
+  font-size: 20px;
+  padding: 0px 60px 0px 60px;
+}
+
+  
+ /* ticket information */ 
+.ticket-container {
+    width: 30%;
+    padding: 15px 25px 35px 0px; /*size*/
+}
+
+.ticket-label {
+    border: none;
+    color: white;
+    padding: 15px 60px;
+    text-align: left;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+	background-color: #bc3434;
+}
+
+.price{
```

Comment:
Try to be more verbose on classes/ids: `tickets-price-label`  or something like that.